lib/hpcbase.pm

```diff
@@ -380,6 +380,8 @@ sub uninstall_spack_module {
 
   get_compute_nodes_deps($mpi)
 
+Obsolete function. Not in use since sle15sp5
```

Comment:
only comment I have is -
why not remove the function itself if is not being called in any test module.

Reply:
i was not sure if i want to remove it because it works for <sle15sp5 and i was thinking it might be useful somehow if i fix it later on.

Reply:
okay, may be we should add this info also along with obsolete function.

Reply:
updated